index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Image slide show</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <div class="slider">
+            <div class="slide slide1">
+                <div class="caption">
+                    <h2>Hayabusa</h2>
+                    <p>Hayabusa can get away very well than other assassins.Basic damage from large hayabusa, this hero is suitable for use as an offlaner and pusher.Cooldown skill that is long enough when used. Does not have a high Attack Speed, and is very skill dependent.</p>
+                </div>
+            </div>
+            <div class="slide slide2">
+                <div class="caption">
+                    <h2>Alucard</h2>
+                    <p>Mobile Legends Demon Hunter, Alucard, mixes both a fighter and assassin playstyle to combine heavy offense with decent durability. Alucard is most often played in a side lane.</p>
+                </div>
+            </div>
+            <div class="slide slide3">
+                <div class="caption">
+                    <h2>Johnson</h2>
+                    <p>MOONTON presents Johnson as the only tank who has mobility as his ability to roam and make his presence in perfect time. Moreover, he can bring one of his teammates to engage in a surprise attack and initiate a war. In this Mobile Legends guide, we will take a look at the best emblem, spell, and builds, including tips and tricks to dominate every game with Johnson.</p>
+                </div>
+            </div>
+            <div class="slide slide4">
+                <div class="caption">
+                    <h2>Claude</h2>
+                    <p>Claude benefits most from the Assassin emblem page, with upgrades in Agility and Invasion giving him added movement speed and physical penetration. You can opt for Bravery over Agility against team compositions where kiting enemy tanks wont be hard, gaining some extra base damage.</p>
+                </div>
+            </div>
+            <div class="slide slide5">
+                <div class="caption">
+                    <h2>Roger</h2>
+                    <p>Roger can freely switch between human and wolf forms. In human form, Roger's Basic Attacks inflict 10% slow on hit. In wolf form, Roger's Basic Attack deal 5% of the target's lost HP as extra [physical] damage (up to 60 against creeps).</p>
+                </div>
+            </div>
+        </div>
```

Comment:
Add one more slide at the end so you may get an infinte scrolling effect.
```suggestion
        <div class="slide slide1">
                <div class="caption">
                    <h2>Hayabusa</h2>
                    <p>Hayabusa can get away very well than other assassins.Basic damage from large hayabusa, this hero is suitable for use as an offlaner and pusher.Cooldown skill that is long enough when used. Does not have a high Attack Speed, and is very skill dependent.</p>
                </div>
            </div>
```

Reply:
i just try it nothing happens on adding one more slide...Animation look same as it was...at the last of slide, animation directly reverse to 1st slide .